Build/ProjectTypes/WebLibrary.props

```diff
@@ -21,4 +21,12 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="$(SolutionDir)Remotion\Web\Development.Analyzers\Web.Development.Analyzers.csproj"
+                      PrivateAssets="all"
+                      ReferenceOutputAssembly="false"
+                      SetTargetFramework="TargetFramework=netstandard2.0"
```

Comment:
Is this done for the common denomitor because we have to use separate entries for ProjectReference otherwise?

Reply:
Yes

Reply:
gut feeling tells me, this isn't perfect. We're not excercising the code during the test that we're using in "production". Since we can't test net48 variant anyway and we also want NRT, there's probably no way around this. Or can we set net5.0 here, too, and have the analyzer run for both net48 and net5.0 compilations?

Reply:
Using an analyzer targeting net5.0 did not work for me. An exception was thrown when creating the analyzer instance.